src/Form/AdminSettingsForm.php

```diff
@@ -387,6 +387,21 @@ private function buildAdvancedConfigurationForm() {
       '#options' => ['trusted' => t('Trusted'), 'untrusted' => t('Untrusted')],
     ];
 
+    $form['inspector_status'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Enable Lift Inspector'),
+      '#default_value' => $settings['inspector_status'],
+      '#description' => t('Enabling this feature will allow you to use type "cntl + i" to get lift information about the session'),
+    ];
+    $form['profile_manager_url'] = [
```

Comment:
Are you sure we can't "guess" this instead of asking the user to fill in this setting? I'll ask the team and get back to you on this.